src/modules/map/components/openlayers/utils/useMapInteractions.composable.js

```diff
@@ -212,7 +212,8 @@ export default function useMapInteractions(map) {
             // triggering a long click on the same spot after 500ms, so that mobile cas have access to the
             // LocationPopup by touching the same-ish spot for 500ms
             longClickTimeout = setTimeout(() => {
-                if (!mapHasMoved) {
+                // we need to ensure long mouse clicks don't trigger this.
+                if (!mapHasMoved && event.originalEvent.pointerType === 'touch') {
```

Comment:
sadly this blocks iOs touch too... 
in the doc [here](https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/pointerType) it says that browser not supporting this feature will fill the `pointerType` with `""`, we could maybe try to react to this value too (maybe that will add back iOs...)
So something like
```Javascript
if (!mapHasMoved && ['' /* iOs support */, 'touch'].includes(event.originalEvent.pointerType)) {
    // ...
}
```

Reply:
or do the negative test
```Javascript
if (!mapHasMoved && event.originalEvent.pointerType !== 'mouse') {
    // ...
}
```

Reply:
the negative seems simpler and more explicit in my opinion. But yeah, I did not think that IoS would be causing problems in that manner